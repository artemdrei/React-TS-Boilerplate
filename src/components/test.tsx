import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Wrapper from '.';

describe('Sample test', () => {
  const wrapper = shallow(<Wrapper />);

  it('Should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
