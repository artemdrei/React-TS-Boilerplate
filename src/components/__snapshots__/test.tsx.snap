// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample test Should match the snapshot 1`] = `
<div>
  <svgReactLoader />
  <Link
    to="/todos"
  >
    Todos
  </Link>
  <Link
    to="/photos"
  >
    Photos
  </Link>
</div>
`;
